lib/chart/animated_line_chart.dart

```diff
@@ -330,6 +367,12 @@ class ChartPainter extends CustomPainter {
     _drawLines(size, canvas);
     _drawAxisValues(canvas, size);
 
+    if (showMarkerLines! &&
```

Comment:
Is there a default value for `showMarkerLines`

Reply:
Yes, it is set to false in ChartPainter class. Should it also be set to false in all the other classes where it is being referenced or is it enough to only set a default value in the last class it is being referenced/set?